azurerm/internal/services/managementgroup/tests/data_source_management_group_test.go

```diff
@@ -37,7 +37,7 @@ resource "azurerm_management_group" "test" {
 }
 
 data "azurerm_management_group" "test" {
-  group_id = azurerm_management_group.test.group_id
+  group_name = azurerm_management_group.test.group_name
```

Comment:
i think this should be
```suggestion
  name = azurerm_management_group.test.name
```

Reply:
Oops... forgot this. Fixed.